jax/experimental/jax2tf/tests/primitive_harness.py

```diff
@@ -1106,7 +1107,11 @@ def _make_conv_harness(name, *, lhs_shape=(2, 3, 9, 10), rhs_shape=(3, 3, 4, 5),
                        padding=((0, 0), (0, 0)), lhs_dilation=(1, 1),
                        rhs_dilation=(1, 1), feature_group_count=1,
                        dimension_numbers=("NCHW", "OIHW", "NCHW"),
-                       batch_group_count=1):
+                       batch_group_count=1, enable_xla=True,
+                       expect_xla_disabled_exception=False):
+  expect_xla_disabled_exception = expect_xla_disabled_exception or (
+      not (enable_xla or jax2tf.jax2tf.ENABLE_TF_CONVOLUTION))
```

Comment:
I was assuming that if we disable XLA we should enable `jax2tf.jax2tf.ENABLE_TF_CONVOLUTION`. I think that there were some cases when it was failing though

Reply:
Yes, we should, but if you remember there was some weird transposition happening that we unsuccessfully tried to debug for a while. I can enable it and you can test internally!